content/eventos/2020-02-27-intro-python-django-breaking-the-enigma-what-if-turing-had-python/contents.lr

```diff
@@ -1,20 +1,49 @@
+information:
+
+Agenda:
+* 6:30-7:00pm - Llegada, networking
+* 7:00-7:40pm - Charla 1
+* 7:40-8:00pm - Receso, pizza
+* 8:00-8:40pm - Talk 2
+* 8:40-9:00pm - Cierre
 ---
 featured_photo: https://secure.meetupstatic.com/photos/event/4/4/d/c/600_489137628.jpeg
 ---
 venue: Koombea
 ---
 address_1: Calle 85 # 53-14
 ---
-talks: 
+talks:
 
 #### talk ####
-title: Intro python & django + Breaking the enigma: what if Turing had python? 
+title: Intro python & django
+----
+details: En esta charla quiero hacer una introducción sobre el lenguaje Python y Django, uno de los frameworks mas populares en el desarrollo web. Cuáles son sus principales características y ventajas que lo hacen ideal para crear aplicaciones monolíticas robustas.
+----
+speaker: javier-daza
+----
+speaker_name: 
+----
+speaker_bio: Ingeniero electrónico de la Universidad del Norte, fundador de Python Barranquilla y desarrollador de Python con más de 4 años de experiencia.
+----
+attach: Systems and computer engineer from Universidad de los Andes, also physicist from the same university with an emphasis in computational astronomy. She is very passionate learner and I enjoy spreading the knowledge to create a stronger community interested in science (both astronomical and computational)
```

Comment:
Esta información es la "speaker_bio" de Maria Camila

_Systems and computer engineer from Universidad de los Andes, also physicist from the same university with an emphasis in computational astronomy. She is very passionate learner and I enjoy spreading the knowledge to create a stronger community interested in science (both astronomical and computational)_

Reply:
Si, en el evento estaba en inglés

Reply:
Ándale, se me pasó, buena esa